src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -23,7 +23,7 @@ export class RulesGrid extends Component {
 
     // TODO: This is a temporary example implementation. Remove with a better version
     this.contextBtns = {
-      disable: <Btn key="disable">Disable</Btn>,
+      disable: <Btn key="disable" onClick={() => console.log('Disabled!')}>Disable</Btn>,
       edit: <Btn key="edit">Edit</Btn>
```

Comment:
Should `Edit` be in translation as well?

Reply:
Same as above